.github/workflows/update-flake.yml

```diff
@@ -22,7 +22,7 @@ jobs:
       
       - uses: cachix/cachix-action@v10
         with:
-          name: sequent-tech
+          name: sequenttech
```

Comment:
@Findeton we should be using either sequent-tech or sequentech, but not sequenttech because our domain name is sequentech.io and even our github's organization name is sequentech, so we should be using that as our standard.